src/stories/5-custom/array-field/array-field.stories.tsx

```diff
@@ -141,11 +141,14 @@ const SCHEMA: Record<string, TFrontendEngineFieldSchema> = {
 const SCHEMA_NESTED_ARRAY: Record<string, TFrontendEngineFieldSchema> = {
 	grid: {
 		uiType: "grid",
-		style: { marginTop: 16, marginBottom: 16 },
+		style: { marginTop: 16, marginBottom: 16, border: "2px solid black", padding: 16 },
```

Comment:
remove styling as it is not needed

Reply:
this is not removed?

Reply:
remove border and padding
```suggestion
		style: { marginTop: 16, marginBottom: 16 },
```